docs/Producing.md

```diff
@@ -1,6 +1,6 @@
 # Producing messages
 
-Producing can be done by creating a `.kafka` file. Write simple producers using the following format:
+Producing can be done by creating a [.kafka](KafkaFile.md#kafkafile) file. Write simple producers using the following format:

```

Comment:
Define simple producers in a [.kafka](KafkaFile.md#kafkafile) file, using the following format:

Reply:
fixed